config/middlewares/isLoggedIn.js

```diff
@@ -0,0 +1,18 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+
+exports.isLoggedIn = (req, res, next) => {
+  const token = req.body.token || req.query.token || req.headers['x-access-token'] || req.headers.authorization;
+  jwt.verify(token, process.env.SECRET, (err, decoded) => {
+    if (err) {
+      return res.status(401).json({
+        message: 'User not logged in',
```

Comment:
Message for 401 status code shouldn't be `'User not logged in'`. @OlatunjiYso @Rhotimee

Reply:
@fortunee, I used 401 because it states that the user is not authenticated which implies that user is not logged in.

Reply:
Oh yeah, that's right! My bad! 😆